### PR TITLE
Added grants to website

### PR DESCRIPTION
**Pull request by tylershuster**
For further review with @jalehman

### CODE REVIEW

**Comment by tylershuster** [comment]
Not sure why the CI preview isn't working; anyway except for the `post_type`  view, this is pretty much in a reviewable state

**Comment by jalehman** [comment]
Screen Shot 2021-04-22 at 8 51 40 AM

😂

**Comment by tylershuster** [comment]
This also needs some mobile responsivity work

File: content/community/grants-and-bounties/-smol-future-sustained-knowledge-sharing-and-marketing.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+[![smol vibe](https://i.gyazo.com/2346666e5b40623353fee1dca42f9879.jpg)](https://i.gyazo.com/2346666e5b40623353fee1dca42f9879.jpg) 
+
+
+
+
+
+
+
+
+
```
Comment (jalehman) [inline content/community/grants-and-bounties/-smol-future-sustained-knowledge-sharing-and-marketing.md]: Why so much whitespace?

File: content/community/grants-and-bounties/thread-documentation.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+[taxonomies]
+grant_type = ["bounty"]
+grant_category = ["Documentation"]
+[extra]
+image = ""
+description = "Document, explain, and give simple howtos for the usage patterns of threads in the Dojo and Gall agents."
+reward = 1
+assignee = "~tinnus-napbus"
+id = "4948904"
+completed = false
```
Comment (jalehman) [inline content/community/grants-and-bounties/thread-documentation.md]:
```suggestion
completed = true
```

File: content/community/grants-and-bounties/bitcoin-full-node-provider-and-wallet.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+[taxonomies]
+grant_type = ["bounty"]
+grant_category = ["App Dev: Arvo"]
+[extra]
+image = ""
+description = "This bounty covers the creation of infrastructure (gall agents) required to enable a Bitcoin-based payments system from within Landscape."
+reward = 6
+assignee = "~timluc-miptev"
+id = "2056919898"
+completed = false
```
Comment (jalehman) [inline content/community/grants-and-bounties/bitcoin-full-node-provider-and-wallet.md]:
```suggestion
completed = true
```

File: content/community/grants-and-bounties/bitcoin-key-derivation.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+[taxonomies]
+grant_type = ["bounty"]
+grant_category = ["App Dev: Other"]
+[extra]
+image = ""
+description = "The Urbit HD Wallet allows for many different keys to be generated from a single master ticket. Your task is to use this mechanism to generate Bitcoin keys and expose ways of interacting with them."
+reward = 2
+assignee = "pkova"
+id = "794126751"
+completed = false
```
Comment (jalehman) [inline content/community/grants-and-bounties/bitcoin-key-derivation.md]:
```suggestion
completed = true
```

File: content/community/grants-and-bounties/haskell-airlock-library.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+[taxonomies]
+grant_type = ["bounty"]
+grant_category = ["Core Dev"]
+[extra]
+image = ""
+description = "Create a library for the Haskell language to be used for Airlock, the API suite for outside communication with Urbit."
+reward = 3
+assignee = "bsima"
+id = "688875191"
+completed = false
```
Comment (jalehman) [inline content/community/grants-and-bounties/haskell-airlock-library.md]:
```suggestion
completed = true
```

File: content/community/grants-and-bounties/staff-the-urbit-help-desk-1.md [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+[taxonomies]
+grant_type = ["bounty"]
+grant_category = ["Community"]
+[extra]
+image = ""
+description = "Help Urbit users with their various problems."
+reward = 1
+assignee = "nartes-fasrum"
+id = "1395801018"
+completed = false
```
Comment (jalehman) [inline content/community/grants-and-bounties/staff-the-urbit-help-desk-1.md]:
```suggestion
completed = true
```

File: content/community/grants-and-bounties/staff-the-urbit-help-desk-2.md [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+[taxonomies]
+grant_type = ["bounty"]
+grant_category = ["Community"]
+[extra]
+image = ""
+description = "Help Urbit users with their various problems."
+reward = 1
+assignee = "sicbud (and i cannot stress this enough) hathes"
+id = "612547183"
+completed = false
```
Comment (jalehman) [inline content/community/grants-and-bounties/staff-the-urbit-help-desk-2.md]:
```suggestion
completed = true
```

File: content/community/grants-and-bounties/staff-the-urbit-helpdesk-east.md [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+[taxonomies]
+grant_type = ["bounty"]
+grant_category = ["Community"]
+[extra]
+image = ""
+description = "Help guide newcomers onto the network."
+reward = 1
+assignee = "~libmud-bonted"
+id = "1140084926"
+completed = false
```
Comment (jalehman) [inline content/community/grants-and-bounties/staff-the-urbit-helpdesk-east.md]:
```suggestion
completed = true
```

File: content/community/grants-and-bounties/staff-the-urbit-helpdesk-west.md [hunk 9]
```diff
@@ -0,0 +1,57 @@
...
+[taxonomies]
+grant_type = ["bounty"]
+grant_category = ["Community"]
+[extra]
+image = ""
+description = "Help guide newcomers onto the network."
+reward = 1
+assignee = "Conrad"
+id = "123890561"
+completed = false
```
Comment (jalehman) [inline content/community/grants-and-bounties/staff-the-urbit-helpdesk-west.md]:
```suggestion
completed = true
```

File: content/community/grants-and-bounties/the-complete-guide-to-gall-and-landscape.md [hunk 10]
```diff
@@ -0,0 +1,77 @@
++++
+title = "The Complete Guide to Gall and Landscape"
+date = 2020-06-22
+[taxonomies]
+grant_type = ["proposal"]
+grant_category = ["Documentation"]
+[extra]
+image = ""
+description = "A full text and video course on every aspect of creating Gall and Landscape apps. Students will create their own full application to finish."
+reward = 2
```
Comment (jalehman) [inline content/community/grants-and-bounties/the-complete-guide-to-gall-and-landscape.md]:
```suggestion
reward = 4
```

File: content/community/grants-and-bounties/the-complete-guide-to-gall-and-landscape.md [hunk 11]
```diff
@@ -0,0 +1,77 @@
...
+assignee = "~timluc-miptev"
+id = "2061597290"
+completed = false
```
Comment (jalehman) [inline content/community/grants-and-bounties/the-complete-guide-to-gall-and-landscape.md]:
```suggestion
completed = true
```

File: content/community/grants-and-bounties/typescript-airlock-library.md [hunk 12]
```diff
@@ -0,0 +1,44 @@
...
+[taxonomies]
+grant_type = ["bounty"]
+grant_category = ["Core Dev"]
+[extra]
+image = ""
+description = "Create a library for the Typescript language to be used for Airlock, the API suite for outside communication with Urbit."
+reward = 3
+assignee = "~radbur-sivmus"
+id = "1845001687"
+completed = false
```
Comment (jalehman) [inline content/community/grants-and-bounties/typescript-airlock-library.md]:
```suggestion
completed = true
```

File: content/community/grants-and-bounties/up-to-date-hoon-reference-card.md [hunk 13]
```diff
@@ -0,0 +1,30 @@
...
+[taxonomies]
+grant_type = ["proposal"]
+grant_category = ["Documentation"]
+[extra]
+image = ""
+description = "I have made an up-to-date version of the Urbit Hoon reference card and will maintain it.https://github.com/natareo/hooncard"
+reward = 1
+assignee = "N E Davis"
+id = "607772785"
+completed = false
```
Comment (jalehman) [inline content/community/grants-and-bounties/up-to-date-hoon-reference-card.md]:
```suggestion
completed = true
```

**Review by jalehman — COMMENTED** [review]
A few high-level things:

Let's make the url `urbit.org/grants` rather than `urbit.org/community/grants-and-bounties`. We can still have it be accessible via the community page in terms of content, but top-level routes are better for SEO, and shorter URLs are easier to work with.

Make sure to remove the `content/docs` commit.

File: templates/grants/list.html [hunk 14]
```diff
@@ -0,0 +1,99 @@
...
+  <a href="/community/grants-and-bounties/">Grants</a>
+
+  {% include "partials/navigation-search.html" %}
+
+</nav>
+<section class="full c4-10-lg pt4 lh-copy">
+
+  <h1>Urbit Grants Program</h1>
+
+  <p>Urbit is a community project. While anyone can contribute, we help focus development and reward exceptional contribution through our grants and bounties program.</p>
```
Comment (jalehman) [inline templates/grants/list.html]:
```suggestion
  <p>Urbit is a community project. While anyone can contribute, we help focus development and reward exceptional contribution through our grants program.</p>
```
A bounty is a type of grant.

File: templates/index.html [hunk 15]
```diff
@@ -85,6 +85,32 @@ <h2 class="f5 fw6 mt0 mb1">{{ page.title }}</h2>
...
+    <span class="f3">&gt;90</span>
+    <figcaption class="f5">Grants &amp; Bounties</figcaption>
+  </figure>
+  <a href="/community/grants-and-bounties" class="bb">Learn more here</a>
+</section>
+
+<section class="full lh-copy c7-9-md c7-9-lg mb5">
+  <div class="list mt4">
+    <p><a href="/docs/development/develop" class="bb">Getting Started Guide</a></p>
+    <p><a href="web+urbitgraph://group/~bitbet-bolbel/urbit-community/graph/~darrux-landes/development" class="bb" target="_blank">Development channel on Urbit</a></p>
```
Comment (jalehman) [inline templates/index.html]:
```suggestion
    <p><a href="web+urbitgraph://group/~bitbet-bolbel/urbit-community/graph/~darrux-landes/development" class="bb" target="_blank">Development channel on Landscape</a></p>
```

File: templates/index.html [hunk 16]
```diff
@@ -85,6 +85,32 @@ <h2 class="f5 fw6 mt0 mb1">{{ page.title }}</h2>
   <a href="/events" class="bb">View more events</a>
 </section>
 
+<section class="full lh-copy c2-7-md c3-7-lg mb5">
+  <h1 class="mb3">Grants and Development</h1>
```
Comment (jalehman) [inline templates/index.html]:
```suggestion
  <h1 class="mb3">Grants Program</h1>
```

File: templates/grants/list.html [hunk 17]
```diff
@@ -0,0 +1,99 @@
...
+
+  <p>We reward contributors in four different ways:</p>
+  <ul class='list'>
+    <li class="mb2"><a class="b bb" href="/grant-type/proposal">Proposals</a> from community members</li>
```
Comment (jalehman) [inline templates/grants/list.html]: Per a previous comment, can this URL be `https://urbit.org/grants/proposals`? Or does it have to be `grant-type` per the taxonomy definition?

File: templates/grants/list.html [hunk 18]
```diff
@@ -0,0 +1,99 @@
...
+    <li class="mb2"><a class="b bb" href="/grant-type/bounty">Bounties</a> for specific features</li>
```
Comment (jalehman) [inline templates/grants/list.html]: This would become `/grants/bounties` per the above comment.

File: templates/grants/list.html [hunk 19]
```diff
@@ -0,0 +1,99 @@
...
+    <li class="mb2"><a class="b bb" href="/community/apprenticeships">Apprenticeships</a> undertaken with a mentor</li>
```
Comment (jalehman) [inline templates/grants/list.html]: and this would become `/grants/apprenticeships`

File: templates/grants/list.html [hunk 20]
```diff
@@ -0,0 +1,99 @@
...
+    <li class="mb2"><a class="b bb" href="/post-type/gift">Gifts</a> given post facto for exceptional contributions</li>
```
Comment (jalehman) [inline templates/grants/list.html]: ...and this would become `/grants/gifts`

File: templates/grants/list.html [hunk 21]
```diff
@@ -0,0 +1,99 @@
...
+  <h2 class="mt5">Available and Closed Grants</h2>
+  
+  <ul class="flex flex-wrap mv5">
+    {% set categories = get_taxonomy(kind="grant_category") %}
+    {% for category in categories.items %}
+      <li style="flex: 0 1 15rem;" class="f5 flex items-center"><a href="{{ category.permalink }}" class="bb">{{ category.name }}</a></li>
+    {% endfor %}
+  </ul>
+
+  <p>This is a partial list, check <a href="https://grants.urbit.org" target="_blank" class="bb">grants.urbit.org</a> to see more and apply for a grant or propose your own.</p>
```
Comment (jalehman) [inline templates/grants/list.html]: Remove this, since this will now be a complete list.

**Review by jalehman — CHANGES_REQUESTED** [review]
A few more changes. I'll get real metrics and forms to you within the next couple of days.

**Comment by jalehman** [comment]
Here's the [proposal submission](https://airtable.com/shrCi54rEDxgSZr3z) form. Still working on the Bounty Work Request form and the Request Payment form.

**Comment by jalehman** [comment]
Here's the [invoice submission form](https://airtable.com/shrXXCs1uaxtNSBcg).

**Comment by mikeosborne** [comment]
The "Apply for this grant" links point to the grants website. We are replacing this with airtable forms. There will be a separate URL for each worker request. Can the template be updated to use these?

**Comment by mikeosborne** [comment]
I have a proposal template that I refer to in the proposal submission static page. Where should it live?

**Comment by jalehman** [comment]
@mikeosborne This got merged through a separate branch, so please submit your latest commit as a separate PR against master. Thanks!

**Pull request closed without merging**